src/Component/Footer/footcss.css

```diff
@@ -77,7 +87,7 @@ footer {
 .link,
```

Comment:
We should have a hover effect
just add a simple underline that's enough

Reply:
done